makerdao/governance-portal-v2
# Update supporters section, add shadow delegate icon

### What does this PR do?

Adds more spacing to supporters rows

Removes fixed height on supporters section so that all supporters are in view

### Steps for testing:
1. Go to latest exec on mainnet, see supporters section with shadow delegate icon

### Screenshots (if relevant):
<img width="978" alt="Screen Shot 2022-03-08 at 3 18 58 PM" src="https://user-images.githubusercontent.com/5225766/157257068-2c159d3b-d4ae-4410-a1db-c9ff2b663812.png">

<img width="925" alt="Screen Shot 2022-03-08 at 3 19 03 PM" src="https://user-images.githubusercontent.com/5225766/157257054-42006f73-8f85-446b-aa77-5bc2ba1fefa3.png">
